src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
remove "extends ColorSupplier"
1. you don't use the method from "parent class".
2. there is no logical relation between the 2 classes.